fix: module exports

kiddyuchina opened the pull request.

talltyler commented:
So that you know the newer changes to the package json seem to break the setup defined in this branch that were working in cloudflare in this example. https://github.com/sutandojs/sutando-examples/tree/main/typescript/rest-hono-cf-d1
Changing the package.json back to what is defined in this PR fixes the issue without it cloudflares wrangler seems to want to choose the browser mjs build.

kiddyuchina commented:
> So that you know the newer changes to the package json seem to break the setup defined in this branch that were working in cloudflare in this example. https://github.com/sutandojs/sutando-examples/tree/main/typescript/rest-hono-cf-d1 Changing the package.json back to what is defined in this PR fixes the issue without it cloudflares wrangler seems to want to choose the browser mjs build.

Thank you for bringing this to my attention! I really appreciate you taking the time to report the issue and provide detailed information. I’ve addressed the problem, and it’s now fixed in version 1.7.1. Please feel free to update and let me know if you encounter any further issues.
